src/Tizen.NUI/src/public/BaseComponents/ViewInternal.cs

```diff
@@ -162,12 +168,16 @@ internal float ParentOriginX
             get
             {
                 float temp = 0.0f;
-                GetProperty(View.Property.PARENT_ORIGIN_X).Get(out temp);
+                PropertyValue _parentOriginX = GetProperty(View.Property.PARENT_ORIGIN_X);
+                _parentOriginX?.Get(out temp);
+                _parentOriginX?.Dispose();
                 return temp;
             }
             set
             {
-                SetProperty(View.Property.PARENT_ORIGIN_X, new Tizen.NUI.PropertyValue(value));
+                PropertyValue pVal = new Tizen.NUI.PropertyValue(value);
```

Comment:
How about change "pVal" as "targetValue" or "setValue" ?

https://readlearncode.com/code-and-stuff/clean-code-variables/
: "Use words that can be pronounced and avoid abbreviations"

Reply:
Done